diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/utils/AstNodeUtils.kt

```diff
@@ -167,13 +167,9 @@ fun ASTNode.findChildAfter(afterThisNodeType: IElementType, childNodeType: IElem
  * method that traverses previous nodes until it finds needed node or it finds stop node
  * @return ASTNode?
  */
-fun ASTNode.prevNodeUntilNode(stopNode: IElementType, checkNode: IElementType) : ASTNode? {
-    if (this.treePrev == null || this.treePrev.elementType == stopNode)
-        return null
-    if (this.treePrev.elementType == checkNode)
-        return this.treePrev
-    return this.treePrev.prevNodeUntilNode(stopNode, checkNode)
-}
+fun ASTNode.prevNodeUntilNode(stopNodeType: IElementType, checkNodeType: IElementType) =
+        siblings(false).firstOrNull { it.elementType != stopNodeType && it.elementType == checkNodeType }
```

Comment:
I think logic of this method has changed after last commit: you should add `takeWhile { it.elementType != stopNodeType }` or something like that, if you want your method to return null when first `stopNode` is encountered. Maybe you should add test in `ASTNodeUtilsTest` to describe intended behaviour.